### PR TITLE
Feature/wa 530 display tokens on main screen

### PR DESCRIPTION
**Pull request by sche**

### CODE REVIEW

File: SafeUIKit/SafeAppUI/AppFlow/MainFlow/NewTransaction/TokenNumberFormatter.swift [hunk 1]
```diff
@@ -16,6 +16,15 @@ class TokenNumberFormatter {
         return formatter
     }()
 
+    static func ERC20Token(code: String? = nil, decimals: Int) -> TokenNumberFormatter {
```
Comment (DmitryBespalov) [inline SafeUIKit/SafeAppUI/AppFlow/MainFlow/NewTransaction/TokenNumberFormatter.swift:19]: could you use this function to create `eth` token number formatter var? would be nice!

File: SafeUIKit/SafeAppUI/AppFlow/MainFlow/Tokens/TokensTableViewController.swift [hunk 2]
```diff
@@ -24,48 +19,32 @@ public class TokensTableViewController: UITableViewController {
...
-        formatter.numberStyle = .currency
-        formatter.currencySymbol = currency
-        return formatter.string(from: NSNumber(value: balance))!
-    }
-
-    private func formattedBalance(_ balance: Double, token: String) -> String {
-        formatter.numberStyle = .decimal
-        return formatter.string(from: NSNumber(value: balance))! + " " + token
+    @objc private func update() {
+        tokens = ApplicationServiceRegistry.walletService.tokens()
```
Comment (DmitryBespalov) [inline SafeUIKit/SafeAppUI/AppFlow/MainFlow/Tokens/TokensTableViewController.swift:33]: is it database call or a network call?

Comment (sche) [reply SafeUIKit/SafeAppUI/AppFlow/MainFlow/Tokens/TokensTableViewController.swift:33]: db

File: SafeUIKit/SafeAppUI/AppFlow/MainFlow/NewTransaction/TokenNumberFormatter.swift (hunk 1 above)
Comment (sche) [reply SafeUIKit/SafeAppUI/AppFlow/MainFlow/NewTransaction/TokenNumberFormatter.swift:19]: ok

Comment (sche) [reply SafeUIKit/SafeAppUI/AppFlow/MainFlow/NewTransaction/TokenNumberFormatter.swift:19]: in next pr

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**